test/e2e/storage/drivers/csi.go

```diff
@@ -490,6 +495,11 @@ func InitGcePDCSIDriver() testsuites.TestDriver {
 			StressTestOptions: &testsuites.StressTestOptions{
 				NumPods:     10,
```

Comment:
Should we increase the number of pods here?

Maybe we need the snapshot test to use its own set of options, so it won't impact the volume stress test settings

Reply:
I wanted to rename this to `VolumeStressTestOptions` but it seems like we'd need to mark this as deprecated if users depend on this in their testdriver.yaml files. Maybe we can add some custom serialization logic and log a deprecation warning? What is the timeline for fully deprecating something like this?

Reply:
Maybe just mark this with a TODO and a tracking issue for the time being, adding a custom decoder seems more involved.

Reply:
How about we create a new VolumeSnapshotStressTestOptions?

Reply:
Filed https://github.com/kubernetes/kubernetes/issues/96241 for this work.

Reply:
Agreed, added. I can rename it to that to be even more specific.